src/server.h

```diff
@@ -1236,20 +1236,22 @@ struct sharedObjectsStruct {
 };
 
 /* ZSETs use a specialized version of Skiplists */
+/* ZSETs 使用了一种特殊的跳表 */
 typedef struct zskiplistNode {
-    sds ele;
-    double score;
-    struct zskiplistNode *backward;
+    sds ele; /* 存储字符串类型的数据 */
+    double score; /* 排序时使用的分值 */
```

Comment:
```suggestion
    double score; /* 分数，用于排序时的比较 */
```
改成分数怎么样，因为我在 zset 中都是以分数来翻译了。